### DISCUSSION TITLE
[Support]:  ghcr.io/blakeblackshear/frigate:stable obliterates host default routes

### DISCUSSION THREAD

**Discussion by nathan-stewart** [blakeblackshear/frigate#10043] [General Support]
### Describe the problem you are having

If I do a docker compose using ghcr.io/blakeblackshear/frigate:stable, once it is up and running it will add additional routes to the host, including a default route going to the 

### Version

0.13.2-6476f8a

### Frigate config file

```yaml
n/a
```

### Relevant log output

```shell
Destination     Gateway         Genmask         Flags   MSS Window  irtt Iface
0.0.0.0         0.0.0.0         255.255.255.255 UH        0 0          0 vethcec3548
0.0.0.0         0.0.0.0         0.0.0.0         U         0 0          0 vethcec3548
0.0.0.0         192.168.1.1     0.0.0.0         UG        0 0          0 enp12s0
169.254.0.0     0.0.0.0         255.255.0.0     U         0 0          0 vethcec3548
172.17.0.0      0.0.0.0         255.255.0.0     U         0 0          0 docker0
172.18.0.0      0.0.0.0         255.255.0.0     U         0 0          0 br-eef4c7912261
192.168.1.0     0.0.0.0         255.255.255.0   U         0 0          0 enp12s0
192.168.1.1     0.0.0.0         255.255.255.255 UH        0 0          0 enp12s0
192.168.12.2    192.168.1.1     255.255.255.255 UGH       0 0          0 enp12s0
```

### FFprobe output from your camera

```shell
n/a
```

### Operating system

Debian

### Install method

Docker Compose

### Coral version

PCIe

### Network connection

Wired

### Camera make and model

Besdersec A6

**Comment by blakeblackshear** [comment]
I think what you are seeing is a part of docker's networking setup. Do you see something different when starting an official nginx image?

**Comment by nathan-stewart** [comment]
It is - a broken part I believe but apparently intentionally broken for inexplicable reasons. I was able to work around it by running in host mode but no matter what the network mode it shouldn't overwrite host routes - but that's definitely not a frigate issue.
